pkg/sfu/bwe/bwe.go

```diff
@@ -94,9 +71,8 @@ type BWE interface {
 
 	HandleTWCCFeedback(report *rtcp.TransportLayerCC)
 
-	ProbingStart(expectedBandwidthUsage int64)
-	ProbingEnd(isNotFailing bool, isGoalReached bool)
-	GetProbeStatus() (isValidSignal bool, trend ChannelTrend, lowestEstimate int64, highestEstimate int64)
+	ProbeClusterStarting(pci ccutils.ProbeClusterInfo)
+	ProbeClusterDone(pci ccutils.ProbeClusterInfo) (bool, int64)
```

Comment:
This method returns `isCongestionClearing, estimatedAvailableChannelCapacity` , signals indicating what happened during the probing phase. `StreamAllocator` will use it to decide whether to up allocate some deficient tracks or discard this probe result.